packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/ToolbarActionWidget.tsx

```diff
@@ -213,12 +254,20 @@ export const ToolbarActionWidget = ({
       />
       <Button
         data-testid={toolbarAction.label}
-        classes={{ root: classes.style }}
+        classes={state.selected ? { root: classes.selected } : { root: classes.style }}
         variant="contained"
-        color={state.selected ? 'secondary' : 'primary'}
         onFocus={() => setState({ message: state.message, selected: true })}
         onBlur={() => setState({ message: state.message, selected: false })}
         ref={ref}>
+        {toolbarAction.imageURL?.length > 0 ? (
+          <img
```

Comment:
Same as for the Image widget, I can't get the image preview to work even on a correctly congfigured toolbar action (I see the image on a an actual instance).

Reply:
Done